docs/tutorials/text_prediction/customization.md

```diff
@@ -1,274 +1,204 @@
 ```
 
-We can now call `fit()` with hyperparameter-tuning over our custom search space. 
-Below `num_trials` controls the maximal number of different hyperparameter configurations for which AutoGluon will train models (5 models are trained under different hyperparameter configurations in this case). To achieve good performance in your applications, you should use larger values of `num_trials`, which may identify superior hyperparameter values but will require longer runtimes.
 
 ```{.python .input}
-predictor_mrpc = task.fit(train_data,
-                          label='label',
-                          hyperparameters=hyperparameters,
-                          num_trials=2,  # increase this to achieve good performance in your applications
-                          time_limits=60 * 2,
-                          ngpus_per_trial=1,
-                          seed=123,
-                          output_directory='./ag_mrpc_random_search')
+predictor.evaluate(dev_data, metrics=['f1', 'acc'])
 ```
 
-We can again evaluate our model's performance on separate test data.
+To visualize the pre-registered hyperparameters, you can call `ag_text_presets.create(key_name)`, e.g.,
+
 
 ```{.python .input}
-dev_score = predictor_mrpc.evaluate(dev_data, metrics=['acc', 'f1'])
-print('Best Config = {}'.format(predictor_mrpc.results['best_config']))
-print('Total Time = {}s'.format(predictor_mrpc.results['total_time']))
-print('Accuracy = {:.2f}%'.format(dev_score['acc'] * 100))
-print('F1 = {:.2f}%'.format(dev_score['f1'] * 100))
+ag_text_presets.create('electra_small_fuse_late')
 ```
 
-And also use the model to predict whether new sentence pairs are paraphrases of each other or not.
+Another way to specify customized config is to directly specify the `hyperparameters` argument in `predict.fit()`. Following is an example
+
 
 ```{.python .input}
-sentence1 = 'It is simple to solve NLP problems with AutoGluon.'
-sentence2 = 'With AutoGluon, it is easy to solve NLP problems.'
-sentence3 = 'AutoGluon gives you a very bad user experience for solving NLP problems.'
-prediction1 = predictor_mrpc.predict({'sentence1': [sentence1], 'sentence2': [sentence2]})
-prediction1_prob = predictor_mrpc.predict_proba({'sentence1': [sentence1], 'sentence2': [sentence2]})
-print('A = "{}"'.format(sentence1))
-print('B = "{}"'.format(sentence2))
-print('Prediction = "{}"'.format(prediction1[0] == 1))
-print('Prob = "{}"'.format(prediction1_prob[0]))
-print('')
-prediction2 = predictor_mrpc.predict({'sentence1': [sentence1], 'sentence2': [sentence3]})
-prediction2_prob = predictor_mrpc.predict_proba({'sentence1': [sentence1], 'sentence2': [sentence3]})
-print('A = "{}"'.format(sentence1))
-print('B = "{}"'.format(sentence3))
-print('Prediction = "{}"'.format(prediction2[0] == 1))
-print('Prob = "{}"'.format(prediction2_prob[0]))
+predictor.fit(train_data, hyperparameters=ag_text_presets.create('electra_small_fuse_late'),
```

Comment:
can you show more customization here? Eg. specifying the learning-rate or regularization

Reply:
I tried to update both number of training epochs and lr.